eessi_bot_event_handler.py

```diff
@@ -599,6 +601,61 @@ def start(self, app, port=3000):
         self.log(log_file_info)
         waitress.serve(app, listen='*:%s' % port)
 
+    def handle_pull_request_merged_event(self, event_info, pr):
+        """
+        Handle events of type pull_request with the action merged. Main action
+        is to scan directories used and move them to the trash_bin.
+
+        Args:
+        event_info (dict): event received by event_handler
+        pr (github.PullRequest.PullRequest): instance representing the pull request
+
+        Returns:
+        github.IssueComment.IssueComment instance or None (note, github refers to
+        PyGithub, not the github from the internal connections module)
+        """
+
+        # Detect event and only act if PR is merged
+        request_body = event_info['raw_request_body']
+        action = request_body['action']
+
+        if action == 'merged':
```

Comment:
Maybe we should verify if this would also apply to PRs that get closed?

Reply:
Handled on 435bcae, will test it now

Reply:
> Handled on [435bcae](https://github.com/EESSI/eessi-bot-software-layer/commit/435bcae9e0f1027489d27470f0e4b6a01412055f), will test it now

Seems to work!